lib/service_check.pm

```diff
@@ -54,11 +55,12 @@ our $default_services = {
         support_ver   => '12-SP3,12-SP4,12-SP5,15,15-SP1'
     },
     # Quick hack for poo 50576, we need this workround before full solution
-    #    apache => {
-    #    srv_pkg_name  => 'apache2',
-    #    srv_proc_name => 'apache2',
-    #    support_ver   => '12-SP3,12-SP4,15,15-SP1'
-    #},
+    apache => {
+        srv_pkg_name       => 'apache2',
+        srv_proc_name      => 'apache2',
+        support_ver        => '12-SP3,12-SP4,15,15-SP1',
```

Comment:
Shall we enable it on 12-SP5? 

As a future improvement, we could get the support_ver from OpenQA variables, since everytime we want to test a new version we will have to amend the code here, which is not scalable for maintaining all the services using this approach.

Reply:
@jlausuch 
1) enable it on 12-SP5 is another PR. https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7828
2) good idea for improvement, but currently i suggest keep current implementation. At the same time i create new ticket into our backlog https://progress.opensuse.org/issues/53987

Reopen it if you do not agree:)